homeassistant/components/switcher_kis/light.py

```diff
@@ -79,32 +77,6 @@ def is_on(self) -> bool:
         data = cast(SwitcherLight, self.coordinator.data)
         return bool(data.light[self._light_id] == DeviceState.ON)
 
-    async def _async_call_api(self, api: str, *args: Any) -> None:
-        """Call Switcher API."""
-        _LOGGER.debug("Calling api for %s, api: '%s', args: %s", self.name, api, args)
```

Comment:
Cleanup the unused `_LOGGER` (`_LOGGER = logging.getLogger(__name__)`)

Reply:
it is done.